content/writing/newsletter.md

```diff
@@ -9,6 +9,32 @@ Capitalize the first word except for conjunctions and try to keep the headers sh
 
 ## Body
 Gmail clips emails that have a message size larger than 102KB (source: [Mailchimp](https://mailchimp.com/help/gmail-is-clipping-my-email/)), so we need to limit the content to what is the most important. We should also adapt our content in a excerpt pointing on our [blog](https://blog.mindee.com), video on [YouTube](https://www.youtube.com/channel/UCXcb0H4P81RqvvvFfWdszoA) or to our [documentation](https://developers.mindee.com/docs) for more information.
+Here is a guided format for the newsletter content
+- Subject heading
+- Introduction
+- Highlight information(if any)
+- Product updates(arranged based on products)
+- Other fun stuff(if any)
+- Videos/Presentations(if any)
+- Latest Mindee articles
+- Article in developers world
+
 
 ## Images
 All images need to be optimize using [Shortpixel](https://shortpixel.com/online-image-compression) glossy compression level to reduce the size of the newsletter.
+
+## Spacing
+The newsletter content should be properly spaced. Add a spacer at the end of a paragraph.
+
+## Color
+Mindee has a set of defined colors that can be used in the newsletter. For example:
+- RED
+ 
+	RGB: `253 50 70`
+  
+	Hexa: `#FD3426`
+
+A complete list of the colors can be found here: [Mindee Colors](https://docs.google.com/document/d/1Myb2dETLb4QcsVZU7hlKi4ZwSFNuKLN8nexG7Qs__Go/)
+
+## Buttons
+Buttons may be included in the newsletter, but they must adhere to the standard button design, which has a border radius of `4px` and the color `#FD3426`.
```

Comment:
I would add a note that it should be limited to the main CTA in the newsletter, everything is should be "read more" links.